test/distributed/test_c10d_error_logger.py

```diff
@@ -107,35 +47,31 @@ def failed_broadcast_not_raise_exception(self):
         except Exception as exception:
             pass
 
-    @with_comms
     def test_exception_handler_with_dist(self) -> None:
+        print(dist.get_rank())
         with self.assertRaises(Exception) as exception:
             self.failed_broadcast_raise_exception()
 
-        with self.assertLogs(dist._c10d_error_logger, level="DEBUG") as captured:
+        with self.assertLogs(
+            dist._c10d_error_logger, level="DEBUG"
+        ) as captured:
             self.failed_broadcast_not_raise_exception()
             error_msg_dict = json.loads(
-                re.search("({.+})", captured.output[0]).group(0).replace("'", '"')
+                re.search("({.+})", captured.output[0])
+                .group(0)
+                .replace("'", '"')
             )
+            print(error_msg_dict)
             self.assertEqual(len(error_msg_dict), 7)
-
-            self.assertIn("func_name", error_msg_dict.keys())
             self.assertEqual("broadcast", error_msg_dict["func_name"])
-
-            self.assertIn("args", error_msg_dict.keys())
-
-            self.assertIn("backend", error_msg_dict.keys())
-            self.assertEqual("nccl", error_msg_dict["backend"])
-
-            self.assertIn("world_size", error_msg_dict.keys())
+            self.assertEqual(f"(tensor([0, 1]), {self.world_size+1}), {{}}", error_msg_dict["args"])
```

Comment:
what's this new assert for?

Reply:
For the exception handler, the logger actually logs the args and kwargs to the collective function. This is the args passed in from the broadcast call above. 

This assert was missing so adding it back here.